tasks/docs.rb

```diff
@@ -189,85 +183,69 @@ def properties_list(properties)
     def special_properties(name, data)
       properties = {}
 
-      properties["common_resource_functionality_multiple_packages"] =
-        case name
-        when "yum_package", "apt_package", "zypper_package", "homebrew_package", "dnf_package", "pacman_package"
-          true
-        when "package"
-          nil
-        else
-          false
-        end
+      # these package properties support passing arrays for the package name
+      properties["common_resource_functionality_multiple_packages"] = true if %w{yum_package apt_package zypper_package homebrew_package dnf_package pacman_package}.include?(name)
 
-      properties["common_resource_functionality_resources_common_windows_security"] = name == "remote_directory"
+      properties["common_resource_functionality_resources_common_windows_security"] = true if name == "remote_directory"
 
-      properties["cookbook_file_specificity"] = name == "cookbook_file"
+      properties["cookbook_file_specificity"] = true if name == "cookbook_file"
```

Comment:
why the tautology here? Are you trying to keep some existing value in these if they are not true?

Reply:
The intention is to avoid creating these yaml entries unless they're set the true. Everything in the hugo config defaults to false and including 30 false values make it really hard to quickly determine what's going on. It gets us cleanup like this https://github.com/chef/chef-web-docs/pull/2476